docs/content/how-to/writing-tables.md

```diff
@@ -262,6 +262,10 @@ filter_spec is equal to the 'WHERE' clause in SQL DELETE statement. Examples:
     id > (SELECT count(*) FROM employee)
 ```
 
+{{< hint info >}}
+If you have set `merge-engine` = `partial-update`, it will be disabled temporarily. 
```

Comment:
I can not get `it will be disabled temporarily`

Reply:
I want to explain that when the actions running, the merge engine settings are ignored; when the actions finishied, the merge engine settings still works.

Reply:
But just dynamic options, no affect to file system.